core/http-auth-aws-crt/src/main/java/software/amazon/awssdk/http/auth/aws/crt/AwsCrtS3V4aHttpSigner.java

```diff
@@ -16,9 +16,24 @@
 package software.amazon.awssdk.http.auth.aws.crt;
 
 import software.amazon.awssdk.annotations.SdkPublicApi;
+import software.amazon.awssdk.http.auth.aws.crt.internal.DefaultAwsCrtS3V4aHttpSigner;
 import software.amazon.awssdk.http.auth.spi.HttpSigner;
+import software.amazon.awssdk.identity.spi.AwsCredentialsIdentity;
 
+/**
+ * An {@link HttpSigner} that will sign a request
+ * using an AWS credentials ({@link AwsCredentialsIdentity}),
+ * specifically for S3-CRT.
+ */
 @SdkPublicApi
-public interface AwsCrtV4aHttpAuthScheme extends HttpSigner {
+public interface AwsCrtS3V4aHttpSigner extends HttpSigner<AwsCredentialsIdentity> {
 
+    /**
+     * Get a default implementation of a {@link AwsCrtS3V4aHttpSigner}
+     *
+     * @return AwsCrtS3V4aHttpSigner
+     */
+    static AwsCrtS3V4aHttpSigner create() {
```

Comment:
I'd like Java team to confirm whether these methods should be called create or defaultSigner?

Reply:
`create()` is fine